### ISSUE TITLE
Crash report alpha-37: NPE in StatsUtils.getCurrentDateTZ

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#5289]
```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.stats.StatsActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.wordpress.android.fluxc.model.SiteModel.getTimezone()' on a null object reference
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2426)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2490)
       at android.app.ActivityThread.-wrap11(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1354)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5443)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.wordpress.android.fluxc.model.SiteModel.getTimezone()' on a null object reference
       at org.wordpress.android.ui.stats.StatsUtils.getCurrentDateTZ(StatsUtils.java:88)
       at org.wordpress.android.ui.stats.StatsActivity.onCreate(StatsActivity.java:173)
       at android.app.Activity.performCreate(Activity.java:6245)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1130)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2379)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2490)
       at android.app.ActivityThread.-wrap11(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1354)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5443)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
```

**Comment by maxme** [comment]
I can reproduce when opening the stats from the Stats Widget (tap on the widget).

**Comment by aforcier** [comment]
Fixed in #5291.
